dotCMS/src/main/webapp/html/portlet/ext/clusterconfig/cluster_config_right_panel.jsp

```diff
@@ -71,9 +76,51 @@
 		</tr>
 		<tr>
 			<td></td>
-			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_es_port") %></td>
-			<td align="right" nowrap>{es.port}</td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_active_shards") %></td>
+			<td align="right">{es.activeShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_relocating_shards") %></td>
+			<td align="right">{es.relocatingShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_initializing_shards") %></td>
+			<td align="right">{es.initializingShards}</td>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)